src/main/java/com/sublinks/sublinksapi/person/entities/PersonRegistrationApplication.java

```diff
@@ -95,6 +94,7 @@ public final boolean equals(Object o) {
   public final int hashCode() {
 
     return this instanceof HibernateProxy ? ((HibernateProxy) this).getHibernateLazyInitializer()
-        .getPersistentClass().hashCode() : getClass().hashCode();
+        .getPersistentClass()
+        .hashCode() : getClass().hashCode();
```

Comment:
Unnecessary, formatting change.

Reply:
It is within our .editorconfig

Reply:
I just mean it's a change to formatting unrelated to the issue.

Reply:
We should run the editor config on every file in some time.